pkg/registry/pod/etcd/etcd_test.go

```diff
@@ -846,6 +847,33 @@ func TestEtcdCreate(t *testing.T) {
 	}
 }
 
+func TestEtcdCreateBindingNoPod(t *testing.T) {
+	registry, bindingRegistry, fakeClient, _ := newStorage(t)
+	ctx := api.NewDefaultContext()
+	fakeClient.TestIndex = true
+
+	key, _ := registry.store.KeyFunc(ctx, "foo")
+	fakeClient.Data[key] = tools.EtcdResponseWithError{
+		R: &etcd.Response{
+			Node: nil,
+		},
+		E: tools.EtcdErrorNotFound,
+	}
+	// Assume that a pod has undergone the following:
+	// - Create (apiserver)
+	// - Schedule (scheduler)
+	// - Delete (apiserver)
+	_, err := bindingRegistry.Create(ctx, &api.Binding{PodID: "foo", Host: "machine", ObjectMeta: api.ObjectMeta{Namespace: api.NamespaceDefault}})
```

Comment:
Just a nit. It might be useful to check if "err" is nil and surface that. Otherwise, LGTM.